rfc/distributed-operation-cache.md

```diff
@@ -0,0 +1,107 @@
+---
+title: "Distributed Operation Cache"
```

Comment:
I'd call this "Cache Warming" or similar.
This is not about caching and it's not distributed,
which might be misunderstood as we're planning to add "response" caching, which serves a completely different purpose.

The purpose of this RFC is solely to warm up all existing caches (normalization, validation, planning)

Reply:
I see where the confusing is coming from. With `distributed` I mean the mechanism is applied to all router instances and each

Reply:
Renamed